geopandas/tests/test_dissolve.py

```diff
@@ -61,7 +61,7 @@ def expected_mean(merged_shapes):
 def test_geom_dissolve(nybb_polydf, first):
     test = nybb_polydf.dissolve("manhattan_bronx")
     assert test.geometry.name == "myshapes"
-    assert test.geom_almost_equals(first).all()
+    assert test.geom_equals_exact(first).all()
```

Comment:
This also needs to be updated to provide the second parameter for `geom_equals_exact`